src/styles/_card.scss

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700;800;900&display=swap");
+
+:root {
+  --star: url(../images/star.svg);
+  --star-active: url(../images/star-active.svg);
+}
+
+.product-card {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 200px;
```

Comment:
you don't need width any more, because you set it in grid-template-columns property

Reply:
if grid will be changed - card will brak up - it is not good